src/components/useList/components/ListItemExpandIcon/ListItemExpandIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import {ChevronDown} from '@gravity-ui/icons';
+
+import {Icon} from '../../../Icon';
+import {colorText} from '../../../Text';
+import {block} from '../../../utils/cn';
+import type {ListItemExpandIconRenderProps} from '../../types';
+
+import './ListItemExpandIcon.scss';
+
+const b = block('list-item-expand-icon');
+
+export interface ListItemExpandIconProps extends ListItemExpandIconRenderProps {}
+
+export const ListItemExpandIcon = ({
+    expanded,
+    disableTransition,
+    usageRole: usage = 'action',
+    disabled,
+}: ListItemExpandIconProps) => {
+    return (
+        <Icon
+            className={b(
+                {expanded, disableTransition, usage},
+                colorText({color: disabled ? 'hint' : undefined}),
+            )}
+            data={ChevronDown}
+            size={16}
+        />
+    );
+};
+
+// For correct rendering inside `Button` component
+ListItemExpandIcon.displayName = 'Icon';
```

Comment:
For correct rendering there is a `Button.Icon` wrapper, everything placed inside would work as an icon

Reply:
This is an undocumented property (`Button.Icon`), how will users know about it?